tgui/packages/tgui/components/TextArea.js

```diff
@@ -0,0 +1,188 @@
+/**
+ * @file
+ * @copyright 2020 Aleksej Komarov
+ * @author Original Aleksej Komarov
+ * @author Changes Warlockd (https://github.com/warlockd)
+ * @license MIT
+ */
+
+
+import { classes, isFalsy } from 'common/react';
+import { Component, createRef } from 'inferno';
+import { Box } from './Box';
+
+
+const toInputValue = value => {
+  if (isFalsy(value)) {
```

Comment:
I'm unfamiliar with JS conventions in SS13, but it seems a bit strange to import a library when you could just use `if(value) {`

Reply:
Its already included in inferno, it basicly checks if its undefined, null or false.

Reply:
>it basicly checks if its undefined, null or false.

So does `if(value) {`, and that isn't dependent upon us ensuring the external library does what it says it does.